lorien/Misc/Settings.gd

```diff
@@ -61,4 +62,60 @@ func set_value(key: String, value = null):
 	_config_file.set_value(DEFAULT_SECTION, key, value)
 	_save_settings()
 	
+# -------------------------------------------------------------------------------------------------
+func _set_default_shortcuts():	
+	for action_name in InputMap.get_actions():
+		for event in InputMap.get_action_list(action_name): 
+			if typeof(event) == TYPE_OBJECT && event.is_class("InputEventJoypadButton"):
+				continue
+			if _config_file.has_section_key("shortcuts", action_name):
+				continue
+			var shortcut = OS.get_scancode_string(event.get_scancode_with_modifiers())
+			_config_file.set_value("shortcuts", action_name, shortcut)
+			continue # for now every action can have only one shortcut
```

Comment:
That would be a problem right now already, because some actions have multiple shortcuts. Also, I think this is a pretty hard requirement for graphics-tablet users that I would like to solve, because I would like to have the possibility to assign button on my tablet AND on the keyboard.

Also, I think the continue does not do what it is intended to do here? I think the intention was a `break` instead to break out of the set_value loop early?!